frontend.rst

```diff
@@ -1,21 +1,40 @@
-Managing CSS and JavaScript
-===========================
-
-.. admonition:: Screencast
-    :class: screencast
-
-    Do you prefer video tutorials? Check out the `Webpack Encore screencast series`_.
-
-Symfony ships with a pure-JavaScript library - called Webpack Encore - that makes
-it a joy to work with CSS and JavaScript. You can use it, use something else, or
-create static CSS and JS files in your ``public/`` directory directly and
-include them in your templates.
+Symfony Frontend Tools
+======================
+
+Symfony ships with two options for handling the CSS and JavaScript in your app:
+
+* :ref:`Webpack Encore <frontend-webpack-encore>` is a powerful tool built with Node
+  on top of `Webpack`_ that allows you to write modern CSS & JavaScript and handle
+  things like JSX (React), Vue or TypeScript. It's the recommended option for
+  new Symfony applications.
+
+* :ref:`AssetMapper <frontend-asset-mapper>`, is a production-ready simpler alternative
+  to Webpack Encore that runs entirely in PHP. It's currently experimental.
+
+==========================================  =================  ===================================
+                                            Encore             AssetMapper
+==========================================  =================  ===================================
+Production Ready?                           yes                yes
+Stable?                                     yes                experimental
+Requirements                                node               none: pure PHP
+Requires a build step?                      yes                no
+Works in all browsers?                      yes                yes
+Supports :doc:`Stimulus/UX </frontend/ux>`  yes                yes
+Supports Sass/Tailwind                      yes                :ref:`yes <asset-mapper-tailwind>`
+Supports JSX, Vue?                          yes                no
+Supports TypeScript                         yes                no
```

Comment:
👍 for the first part!

About "Vue single-file components", this is trickier. You CAN use Vue with AssetMapper, because there is a pure-JavaScript format you can write your Vue components in - https://github.com/symfony/ux/blob/2.x/ux.symfony.com/assets/vue/components/PackageList.js. But, as far as I can tell, there is NO way to compile a a Vue SFC into pure JS. I would LOVE for someone to prove me wrong - I was able to compile both JSX and Svelte SFC without issues. Vue SFC's are a bit trickier, because they can include CSS. That being said, in theory, that CSS could be compiled to JavaScript that injects those styles... which while probably not workable for a SPA, would likely be fine for many asset mapper users that maybe just want to use a little bit of Vue.

Reply:
Sorry for the confusion. What I meant with Vue SFC is that it currently states: "Supports JSX, Vue?"

This is a bit misleading, as Vue SFC is a very specific subset of Vue. The Vue docs describe all concepts before finally introducing SFC in the 24th article of the docs. So we should say "Supports JSX, Vue SFC?", as Vue without SFC is supported (it's just normal JS).

Reply:
I changed to `Supports React, Vue, Svelte?` and set the answer to "yes" with a `**` for more explanation.

Otherwise, I think it was too tricky to answer precisely :). JSX and Vue (non SFC) were "yes, but you need to compile them). And Vue SFC is "no, no matter what". I mention the Vue SFC in the explanation right below.

Reply:
Looking good :+1: